pontoon/sync/changeset.py

```diff
@@ -234,6 +242,18 @@ def execute_obsolete_db(self):
             .filter(pk__in=self.changes['obsolete_db'])
             .update(obsolete=True))
 
+    def execute_obsolete_resources(self):
+        for path in self.changes['obsolete_resources']:
+            for locale in self.db_project.locales.all():
+                file_path = os.path.join(
+                    locale_directory_path(self.vcs_project.checkout_path, locale.code),
+                    path
+                )
+                if os.path.exists(file_path):
+                    log.debug('Removing obsolete file {} for {}.'.format(path, locale.code))
```

Comment:
We might want to shift this to log.info.

I'd also remove all other log messages in this PR.